### ISSUE TITLE
Make epochs and learning rate adjustable.

### ISSUE THREAD

**Issue by Karlo-Emilo** [microbit-foundation/cctd-ml-machine#456]
Now that we have lifted the lid on the training process with the loss graph, there should be some handles students can play with.

To make the loss graph more interactive, epochs and learning rate could be made adjustable with sliders.

**Comment by r59q** [comment]
Fixed as of 3.3
